graphql-schemas/schema.yml

```diff
@@ -1957,6 +1957,21 @@
   - { name: settings, type: VaultAuthSettings_v1 }
   - { name: policy_mappings, type: VaultPolicyMapping_v1, isList: true }
 
+- name: VaultEntityAlias_v1
+  fields:
+  - { name: name, type: string, isRequired: true }
+  - { name: auth_type, type: string, isRequired: true }
+  - { name: custom_metadata, type: json }
+
+- name: VaultEntity_v1
+  fields:
+  - { name: name, type: string, isRequired: true }
+  - { name: type, type: string, isRequired: true }
+  - { name: policies, type: string, isList: true, isRequired: true }
```

Comment:
could this be a list of references to `VaultPolicy_v1` objects as well?

Reply:
I briefly considered this early on but didn't pursue it because I had no need for the rules portion of ```VaultPolicy_v1```. But I ended up needing to write a policy validation check within the logic to ensure listed policies existed.. This wouldn't be necessary if I had used ```VaultPolicy_v1```. Thanks for bringing up, I am going to go back and update the schema/rewrite the logic to handle these objects instead.